client/src/components/CustomButton.tsx

```diff
@@ -67,17 +67,17 @@ const RedirectButton: FC<RedirectButtonProps> = ({
 
   return (
     <Button
-      variant={buttonLabel === 'SIGN IN' ? 'text' : 'contained'}
+      variant={buttonLabel === 'Sign In' ? 'text' : 'outlined'}
       onClick={() => handleRedirect()}
       sx={{
-        color,
-        backgroundColor,
+        color: color,
+        backgroundColor: borderColor,
         fontSize: '1rem',
         fontWeight: '700',
         borderRadius: '10px',
         '&:hover': {
-          backgroundColor: color,
-          color: backgroundColor,
+          borderColor: borderColor,
+          color: borderColor,
```

Comment:
Shouldn't it be like below, or is it intensional?
```suggestion
          borderColor: borderColor,
          color: color,
```

either way I would like it to look like this: 
```suggestion
          borderColor,
          color,
```